common/v2/services/EthService/utils/makeTransaction.ts

```diff
@@ -10,7 +10,14 @@ import { hexEncodeQuantity } from './hexEncode';
 
 export const makeTransaction = (
   t: Partial<Tx> | Partial<ITransaction> | Partial<IHexStrTransaction> | Buffer | string
-) => new Tx(t);
+) => {
+  if (typeof t === 'object' && t.hasOwnProperty('chainId')) {
```

Comment:
If you wish, we now have include `ramda` in the project.
It provides a `R.prop` that also handles if the object is valid. https://github.com/ramda/ramda/tree/v0.26.1/source/prop.js